File: src/main/scala/mffs/item/card/ItemCard.java
package mffs.item.card;

import resonant.api.mffs.card.ICard;
import mffs.base.ItemMFFS;

public class ItemCard extends ItemMFFS implements ICard
{
	public ItemCard(int id, String name)
	{
		super(id, name);
		this.setMaxStackSize(1);
	}
}

File: src/main/scala/mffs/item/card/ItemCard.scala
package mffs.item.card

import com.resonant.core.prefab.modcontent.AutoItemTexture
import mffs.api.card.Card
import mffs.base.CategoryMFFS
import nova.core.item.Item

class ItemCard extends Item with Card with CategoryMFFS with AutoItemTexture {
	override def getMaxCount: Int = 1

	override def getID: String = "cardBlank"
}